src/component/changetracking/AssetHistory.tsx

```diff
@@ -25,7 +27,28 @@ export const AssetHistory: React.FC<AssetHistoryProps> = (props) => {
   const [records, setRecords] = React.useState<null | ChangeRecord[]>(null);
   React.useEffect(() => {
     if (asset.iri !== Constants.EMPTY_ASSET_IRI) {
-      loadHistory(asset).then((recs) => setRecords(recs));
+      //Check if vocabulary/term is a snapshot
+      if (
+        (asset instanceof Term || asset instanceof Vocabulary) &&
+        asset.snapshotOf()
```

Comment:
I thinks `asset.isSnapshot()` would better communicate the intent of the `if` condition

Reply:
I agree but for some reason it is needed to check whether the `asset.snapshotOf()`has some value. It can happen that the asset returns true for `asset.isSnapshot()` but the property containg information about the original asset is missing, which causes a crash. This would happen everytime a user tried to open a term from a snapshotted vocabulary summary.
I agree that the intent could be expressed more clearly by changing the if statement, but it would require change on the loading part (to ensure that the `asset.snapshotOf()` contains a value) and this was just faster to implement. If you think that the `if statement` is really not clear I can look more into it.
@ledsoft

Reply:
Ok, I see. Let's leave it as is.